README.md

```diff
@@ -43,6 +44,33 @@ A
   c: 4
 ```
 
+The macro also supports constructors for named tuples with default values; e.g.
+
+```julia
+julia> MyNT = @with_kw (x = 1, y = "foo", z = :(bar))
+(::#5) (generic function with 2 methods)
+
+julia> MyNT()
+(x = 1, y = "foo", z = :bar)
+```
+
+These constructors can be used as bona fide constructors; e.g. 
+
+```julia
+julia> MyNT(x = 2)
+(x = 2, y = "foo", z = :bar)
+
+julia> MyNT(z = x -> x^3)
+(x = 1, y = "foo", z = #8)
+```
+
+These named tuples can be unpacked exactly like other decorated `struct`s (see below). 
```

Comment:
Not quite.  For structs the `@unpack_MyStruct` macros get defined also.  But not for NTs.

Also mention somewhere, that in Julia 0.6 NameTuples needs to be imported/used.

Reply:
So these can't be unpacked? I thought `@unpack` worked via `getfield`, which holds for NTs as well.

Reply:
Yes, they can be unpacked.  But note that `@unpack` works irrespective of whether `@with_kw` has been used.  The only extra unpacking thingy `@with_kw` makes is the `@unpack_MyStruct`.